Increase verbosity

stephenc87 opened the pull request:
The Il2CppDumper and Il2CppAssemblyUnhollower executables will now display their commandlines in the log files, this will help make debugging easier

stephenc87 committed:
Increase verbosity

The Il2CppDumper and Il2CppAssemblyUnhollower executables will now display their commandlines in the log files, this will help make debugging easier

HerpDerpinstine commented:
Why didn't you just add a logger call inside the base Execute method instead?
It would have saved you so much trouble, been more dynamic, and you could have easily iterated through the array of strings and logged them instead of defining the command line twice for no good reason.

stephenc87 committed:
Print path on execution

Would agree with suggestion, thank you, it does make more sense and it is less complicated

stephenc87 commented:
You are absolutely correct, although it wasn't too much trouble, it does make code look somewhat cleaner, I should of done this to begin with

stephenc87 committed:
include quotation marks
